x-pack/test/functional/apps/security/doc_level_security_roles.ts

```diff
@@ -40,15 +41,12 @@ export default function ({ getService, getPageObjects }) {
             },
           ],
         },
-        kibana: {
```

Comment:
__note:__  I removed this argument since in all usages of `addRole` we specify the same `kibana` value. If we ever need to customize it in the future, we can easily recover this logic.